Scripts/script-ReadPDFFile.yml

```diff
@@ -58,6 +59,13 @@ script: |-
           except PDFEncryptionError:
               mark_suspicious('Possibly encrypted file')
               sys.exit(0)
+          except TypeError:
+              tb = traceback.format_exc()
+              if 'pdf = pdfx.PDFx(args.pdf)' in tb and "out_str = in_str.decode(enc['encoding'])" in tb:
+                  mark_suspicious("The script failed to read the PDF file. This might happen if your file is an image")
```

Comment:
```suggestion
                  mark_suspicious("The script failed to read the PDF file. This might happen if your PDF file contains onlu an image")
```